templates/pricing/desktop.html

```diff
@@ -261,32 +408,27 @@ <h2>Power to developers. Peace of mind for IT. Get Ubuntu Pro Desktop.</h2>
           </li>
           <li class="p-list__item is-ticked">Optional weekday or 24x7 support tiers</li>
         </ul>
-        <p>For more information, download our whitepaper:</p>
-        <p>
+        <div class="p-cta-block">
```

Comment:
This `p-cta-block` is fine since it adds the additional padding (which I think is supposed to be `p-section--shallow`) which matches Design. Lmk if I'm wrong